apis/servicediscovery/v1alpha1/generator-config.yaml

```diff
@@ -0,0 +1,6 @@
+ignore:
+  resource_names:
+    - PublicDNSNamespace
+    - Service
+    - HTTPNamespace
+# How to add +immutable to required fields Region, Name, and VPC
```

Comment:
Comment should be removed

Reply:
How would someone updating the code know that I'd previously added the immutable keyword to required fields in the generated code?